contrib/win32/win32compat/w32fd.c

```diff
@@ -814,7 +814,10 @@ int
 w32_ftruncate(int fd, off_t length) {
     CHECK_FD(fd);
 
-    if (!SetFilePointer(w32_fd_to_handle(fd), length, 0, FILE_BEGIN))
+	LARGE_INTEGER new_postion;
```

Comment:
Move declarations to top of code block (C style, to be consistent with rest of OpenSSH code)